src/test-e2e/po/stepper.po.ts

```diff
@@ -28,7 +28,11 @@ export class StepperComponent extends Component {
   }
 
   next() {
-    return this.nextButton().click();
+    const nextButton = this.nextButton();
+    return browser.controlFlow()
+      // Although locally this might not seem needed it's needed in travis
+      .execute(() => browser.executeScript('arguments[0].scrollIntoView(true)', nextButton.getWebElement()))
+      .then(() => nextButton.click());
```

Comment:
I don't think you need to do .then - nextButton.click will run in the control flow after browser.executeScript